### PR TITLE
Add support for non-function calling LLMs in llm graph transformers

### PR DESCRIPTION
**Pull request by tomasonjo**

### CODE REVIEW

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (jexp) [inline libs/experimental/langchain_experimental/graph_transformers/llm.py:215]: should we add this conditionally to the prompt depending if we have node_labels and rel_types?

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (jexp) [inline libs/experimental/langchain_experimental/graph_transformers/llm.py:514]: where do we append nodes?

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:487]: We can allow that, but it has a predefined output which needs to be followed. We could think about something like additional instructions params that just adds some text to prompt.

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:215]: sure

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:514]: We don't because we don't really need them

**Comment by tomasonjo** [comment]
@baskaryan I've addressed the comments

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:532]: But I added them anyway

**Comment by tomasonjo** [comment]
It would make sense to maybe add "additional_instructions" params, so that users don't have to rewrite the whole prompt, but that can wait for another PR

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (baskaryan) [inline libs/experimental/langchain_experimental/graph_transformers/llm.py:499]: nit: can we give `prompt` a default value of None so we can more rigorously check if user has passed in a value or not?

**Review by baskaryan — COMMENTED** [review]
one nit otherwise lgtm. will let @jexp also approve

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py]: Fixed

**Comment by tomasonjo** [comment]
@jexp addressed all the comments. Waiting for an approval

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (jexp) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:530]: thx, you never know what people will use them for :)

**Review by jexp — APPROVED** [review]
LGTM

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (baskaryan) [inline libs/experimental/langchain_experimental/graph_transformers/llm.py:497]: nit: not needed

**Review by baskaryan — APPROVED** [review]

File: libs/experimental/langchain_experimental/graph_transformers/llm.py
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:497]: I didn't find the json-repair in poetry deps or lock?

**Pull request merged**
